packages/frontend/src/providers/AppProvider.tsx

```diff
@@ -93,6 +94,45 @@ export const AppProvider: FC = ({ children }) => {
         }
     }, [health, isCohereLoaded, user]);
 
+    useEffect(() => {
+        const identify = () => {
+            if (user.data) {
+                (window as any).$chatwoot?.setUser(user.data.userUuid, {
+                    email: user.data.email,
+                    name: `${user.data.firstName} ${user.data.lastName}`,
+                });
+            }
+        };
+        if (
+            !isChatwootLoaded &&
+            health.data &&
+            health.data.chatwoot.websiteToken.length > 0 &&
+            health.data.chatwoot.baseUrl.length > 0
+        ) {
+            (window as any).chatwootSettings = {
+                hideMessageBubble: false,
+                position: 'right',
+                locale: 'en',
+                type: 'standard',
+            };
+            const script = document.createElement('script');
+            const ref = document.getElementsByTagName('script')[0];
+            script.src = `${health.data.chatwoot.baseUrl}/packs/js/sdk.js`;
+            script.defer = true;
+            script.async = true;
+            ref.parentNode?.insertBefore(script, ref);
+            script.onload = () => {
+                (window as any).chatwootSDK.run({
+                    websiteToken: health.data.chatwoot.websiteToken,
+                    baseUrl: health.data.chatwoot.baseUrl,
+                });
+            };
+            setIsChatwootLoaded(true);
+            window.addEventListener('chatwoot:ready', identify);
+        }
+        identify();
```

Comment:
For what scenario do we need to trigger the identify here ? 

- If the user loads a page and is unidentified, we load the chatwoot and later on they login so we identify the user in chatwoot via the TrackingProvider. 
- If the user loads a page and is identified, we load the chatwoot and once we get the 'chatwoot:ready' event we identify the user.

Reply:
This is for the case where:
1. User is identified (but widget not loaded so it's a no-op)
2. Widget loads
3. User is identified again from `Chatwoot:ready` event (now widget has loaded and user is identified)